docs/content/migration.md

```diff
@@ -0,0 +1,535 @@
+element while not affecting other components of that type. For example, one `mat-button` cannot
+use the default theme and then another `mat-button` use `myCustomTheme`. Instead you would need to
+manually apply the desired theme colors and hues using CSS selectors and the
+[`mat-color` mixin][74].
+
+#### Run-time Theming in an Angular Material app
+
+Features include:
+- Angular's [Meta Service][147] can be used to dynamically modify `<meta>` tags that affect browser
+header and status bar colors. The [AngularJS Material docs][149] for this feature have links to
+learn more about the specific `<meta>` tags.
+
+Unsupported features:
+- Changing a component's hues like AngularJS Material's [`md-hue-1` class][169]
+- Definition and generation of themes at run-time
+- Change the application-wide default theme to another theme at run-time
+- Use dynamically changing themes for specific components (overriding the default theme) at run-time
+- Dynamically apply `theme-palette-hue-opacity` color combinations as CSS styles to your own
+  application-specific components
+
+
+### <a name="layout"></a> Changes to Layout Features
+
+The AngularJS Material [layout][108] system includes includes attributes and classes like `layout`,
+`flex`, `show`, `hide`, etc.
+
+#### Angular Layout
+
+This comprehensive layout system was moved into its own project named [Angular Layout][120] 
+(`@angular/flex-layout`). 
+
+The package contains the following:
+- A [Declarative API][116] that includes attributes like `fxLayout`, `fxFlex`, `fxHide`, `fxShow`,
+  and more.
+- A [Responsive API][117] that includes attributes like `fxLayout.lt-md`, `fxFlex.gt-sm`,
+  `fxHide.gt-xs`, `[ngClass.sm]`, `[ngStyle.xs]`, and more.
+- Both APIs include [CSS Grid][65] features in addition to the CSS Flexbox features from
+  AngularJS Material.
+- The ability to define [Custom Breakpoints][118]
+- A [MediaObserver Service][119] that allows subscribing to `MediaQuery` activation changes
+
+The [Angular Layout][120] library's full UMD bundle is around `275` KBs. This size can be reduced by
+consuming the various modules and using the Angular CLI to remove unused library code, but this
+may be more complexity than you need.
+
+#### CDK Layout
+
+The Angular CDK's [Layout][121] module provides:
+- A set of `Breakpoints` that match the [Material Design][122] responsive layout grid breakpoints
+- A `BreakpointObserver` Service that provides
+    - the ability to check media query matching state via `isMatched()`
+    - the ability to subscribe to a stream of events based on changes to an array of media queries
+      that are passed into `observe()`
+- A `MediaMatcher` Service that provides access to the low level, native [`MediaQueryList`][123]
+
+If you are building responsive components or need responsive features in your application with a
+minimal increase in bundle size, then the CDK's Layout module is an option you should consider.
```

Comment:
I would omit this paragraph here and instead move to the introduction of this section something like
```markdown
The CDK's layout capabilities offer a lightweight alternative (or complement) to larger, full-blown layout systems.
```